media-sound/teamspeak-client/files/ts3-qt5.12-symbol-reexports.patch

```diff
@@ -0,0 +1,141 @@
+diff --git a/Makefile b/Makefile
+new file mode 100644
+index 0000000..c036d5f
+--- /dev/null
++++ b/Makefile
+@@ -0,0 +1,2 @@
++overrides.so: qt-5.12-overrides.o override-version.map
++	$(CXX) $(LDFLAGS)  -shared -o $@ -Wl,--version-script override-version.map qt-5.12-overrides.cpp
```

Comment:
Since ```$(CXX)``` is not definied here, it will fallback to ```c++``` during compile. This is not valid ;-) Please add ```inherit toolchain-funcs``` and make a ```tc-export CXX``` as last in ```src_prepare```, so it will use systems ```x86_64-pc-linux-gnu-g++```

Reply:
Will do. Is there any good document I can read about this?

Reply:
Done.

Reply:
Looks good, thanks.
Well, about documents, most of my stuff are from https://devmanual.gentoo.org/ebuild-writing/index.html and directly from devs in irc and in review here.